docs/pim/create_product.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
This whole page seems to be a leftover from https://github.com/ezsystems/user-documentation/pull/166. Maybe you didn't base this PR on master, buton your own branch?